### ISSUE TITLE
[Mod Incompatability]Screen Safe Area

### ISSUE THREAD

**Issue by JustCauseWhyNot** [juancarloscp52/BedrockIfy#279]
I find a lot of mods unfortunately don't respect/adhere to the setting screen safe area. I could list a few mods such as detail armor bar, boathud, and customHud. But I don't know if you can fix this issue for each mod. But this is the only mod that seems to add the screen safe area feature so I come here to report the "bug".

**Comment by Dekamir** [comment]
Does Bedrockify compensate for AppleSkin, or vice versa? It looks like most mods don't recognize the safe area and mod developers generally refuse to support Bedrockify.

**Comment by JustCauseWhyNot** [comment]
AppleSkin has worked fine for me.

**Comment by Dekamir** [comment]
Yeah, that's the reason I specifically asked for AppleSkin as it didn't support this in previous versions (and previous MC versions). Currently it does. Is this fixed on AppleSkin's side or Bedrockify's side?

**Comment by juancarloscp52** [comment]
> Is this fixed on AppleSkin's side or Bedrockify's side?

It is fixed on bedrockify side. Compatiblity of screen safe area with other mods is usually implemented in bedrockIfy explicitly.

**Comment by mineland-2048** [comment]
Apple skin compatibility seems to still be a bit broken as of writing this (v1.9.6 for mc 1.21). The overlay it adds appears to be twice the specified size of the safe area.  
![image](https://github.com/user-attachments/assets/0d2cf97e-c603-42a4-abf1-5def44f9adc1)
![image](https://github.com/user-attachments/assets/211f5cc8-7c9a-42b8-ad90-e111999dd136)

(safe area set to 10 and 1 accordingly)

**Comment by juancarloscp52** [comment]
@mineland-2048 I've fixed AppleSkin compatiblity again in 03b18900ab9f0c5e29341be43b1e6d2572a649c4, now works properly with safe area and hud opacity. It will be published in some days! Thanks for letting me know!
